dev_guide/deployments.adoc

```diff
@@ -829,3 +952,283 @@ spec:
   serviceAccount: <service_account>
   serviceAccountName: <service_account>
 ----
+
+[[advanced-deployment-strategies]]
+== Advanced Deployment Strategies
+
+[[advanced-deployment-strategies-blue-green-deployments]]
+=== Blue-Green Deployment
+
+link:http://martinfowler.com/bliki/BlueGreenDeployment.html[Blue-Green
+deployments] involve running two versions of an application at the same time and
+moving production traffic from the old version to the new version. There are
+several ways to implement a blue-green deployment in {product-title}.
+
+[[advanced-deployment-strategies-when-to-use-blue-green-deployment]]
+=== When to Use a Blue-Green Deployment
+
+Use a Blue-Green deployment When you want to test a new version of your
+application in a production environment before moving traffic to it.
+
+Blue-Green deployments make switching between two different versions of your
```

Comment:
Sometimes it's init capped "Blue-Green" and elsewhere it's lowercase "blue-green". Pick one and sync up on it? I'd lean towards lowercase.